.github/workflows/deploy-integ-appstream-egress.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         with:
           go-version: 1.13
       - name: Install pnpm
-        run: npm install -g pnpm
```

Comment:
why pinning this version? how about upstream working or not?

Reply:
Merge from global repo